Rephrase tooltips for protection and reduction attributes (#11913)
endless-sky/endless-sky: data/_ui/tooltips.txt

	`The number of people (passengers or crew) your ship can hold.`

tip "burn protection:"

> 	`Protection provided against burn damage. Modifies by the given factor how much burn damage a ship can withstand before its heat rises as much as an unprotected ship.`

Reviewer: ```suggestion
	`Protection provided against burn damage. Increases by the given factor how much burn damage a ship can withstand before its heat rises as much as an unprotected ship.`
```

> 	`Protection provided against burning damage. Modifies how difficult a ship is to burn by a percentage equal to the given factor: a total value of 50 allows a ship to withstand 50% more burning damage to be burned as much as an unprotected ship.`

Reviewer: ```suggestion
	`Protection provided against burning damage. Modifies how difficult a ship is to burn by a percentage equal to the given factor: a total value of 50 allows a ship to withstand 50% more burning damage before its heat rises as much as an unprotected ship.`
```

	`This ship is unable to regenerate hull whilst cloaked, instead its hull regeneration will suffer a delay that will grow the longer it is cloaked.`

tip "cloak hull protection:"
	`Protection provided against weapons that do hull damage when the cloak is active. Modifies by the given factor how much hull damage a ship can withstand before it loses as much energy as an unprotected ship.`

tip "cloak shield protection:"

> 	`Protection provided against weapons that do shield damage when the cloak is active. Modifies by the given factor how much shield damage a ship can withstand before it loses as much energy as an unprotected ship.`

Reviewer: ```suggestion
	`Protection provided against weapons that do shield damage when the cloak is active. Increases by the given factor how much shield damage a ship can withstand before it loses as much energy as an unprotected ship.`
```

	`The chance that this outfit will drop as flotsam when the ship it is installed on is destroyed.`

tip "force protection:"

> 	`Protection provided against hit force from weapons. Modifies by the given factor how much external force a ship can withstand before it is moved as much as an unprotected ship.`

Reviewer: ```suggestion
	`Protection provided against hit force from weapons. Increases by the given factor how much external force a ship can withstand before it is moved as much as an unprotected ship.`
```

	`Modifies the percentage of the remaining hull strength at which the ship will become disabled.`

tip "hull protection:"

> 	`Protection provided against direct hull damage. Modifies by the given factor how much hull damage a ship can withstand before it loses as much hull as an unprotected ship.`

Reviewer: ```suggestion
	`Protection provided against direct hull damage. Increases by the given factor how much hull damage a ship can withstand before it loses as much hull as an unprotected ship.`
```

	`Amount of credits received per day when you own this outfit, even if it is not in active use.`
	
tip "inertia reduction:"

> 	`Reduces effective ship mass. Modifies how quickly a ship accelerates by the given factor.`

Reviewer: ```suggestion
	`Reduces effective ship mass. Increases how quickly a ship accelerates by the given factor.`
```

	`Percentage of this weapon's damage that leaks through to the hull while the target's shields are still active.`

tip "piercing protection:"

> 	`Protection provided against piercing weapons. Modifies how difficult a ship's shields are to pierce by the given factor.`

Author: I modified your second suggestion to use the same sentence structure as the others; I think it is more clear that way.

	`The number of seconds it takes for additional shield generation to begin after your shields have been depleted.`

tip "shield protection:"

> 	`Protection provided against shield damage. Modifies by the given factor how much shield damage a ship can withstand before it loses as much shields as an unprotected ship.`

Reviewer: ```suggestion
	`Protection provided against shield damage. Increases by the given factor how much shield damage a ship can withstand before it loses as much shields as an unprotected ship.`
```

	`Prevents a ship from emitting scan sounds.`

tip "slowing protection:"

> 	`Protection provided against slowing damage. Modifies by the given factor how much slowing damage a ship can withstand before it as slowed as an unprotected ship.`

Reviewer: ```suggestion
	`Protection provided against slowing damage. Increases by the given factor how much slowing damage a ship can withstand before it as slowed as an unprotected ship.`
```
